components/tools/OmeroPy/test/integration/clitest/test_import.py

```diff
@@ -153,29 +154,38 @@ def add_client_dir(self):
         client_dir = dist_dir / "lib" / "client"
         self.args += ["--clientdir", client_dir]
 
-    def check_other_output(self, out):
-        """Check the output of the import except Images, Plates and Summary"""
-        assert "Other imported objects:" in out
+    def check_other_output(self, out, import_type='default'):
+        """Check the output of the import except objects
+           (Images, Plates, Filesets) and Summary"""
+
         assert "==> Summary" in out
+        if import_type == 'default' or import_type == 'legacy':
+            assert "Other imported objects:" in out
+        elif import_type == 'yaml':
+            assert "Imported objects:" in out
+        elif import_type == 'legacy':
```

Comment:
You can never get here because it's matched by the earlier `if import_type == 'default' or import_type == 'legacy':`